src/main/java/io/github/moulberry/notenoughupdates/util/HypixelApi.java

```diff
@@ -27,7 +27,7 @@
     private static final int FAILS_BEFORE_SWITCH = 3;
     private int currentUrl = 0;
     private long lastPrimaryUrl = 0;
-    private final String[] myApiURLs = {"https://moulberry.codes/"};//, "http://moulberry.codes/", "http://51.79.51.21/"};//, "http://51.75.78.252/" };
+    private final String[] myApiURLs = {"http://51.79.51.21/"};//, "http://moulberry.codes/", "/"};//, "http://51.75.78.252/" };
```

Comment:
Why did you change this?

Reply:
I was having trouble connecting cause of my isp and I forgot to change it back.